author.php

```diff
@@ -14,8 +14,11 @@
 $context          = Timber::context();
 $context['posts'] = new Timber\PostQuery();
 if ( isset( $wp_query->query_vars['author'] ) ) {
-	$author            = new Timber\User( $wp_query->query_vars['author'] );
-	$context['author'] = $author;
-	$context['title']  = 'Author Archives: ' . $author->name();
+	$factory = new Timber\Factory\UserFactory();
+        $author = $factory->from((int) $wp_query->query_vars['author']);
```

Comment:
Hello!
You may mix space and tab indentations.

Reply:
updated